bee-common-derive/tests/packable/dup_id_enum.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(unused_imports)]
```

Comment:
```suggestion
#![allow(unused_imports)]

```
For this one and all the others :p